### PR TITLE
N14: Oh no, Lava!

### PR DESCRIPTION
**Pull request by troligtvis**
Part of #278 ("Newsletter 14")

### CODE REVIEW

File: content/posts/newsletter-014/index.md [hunk 1]
```diff
@@ -62,6 +62,20 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+### Oh no, Lava!
+
+![oh-no-lava](ohnolava_troligtvis.gif)
+
+Oh no, Lava! is the working title of a platforming game which take inspiration
+from an old iOS game created back in 2014. The game is running with [Bevy][bevy]
+as its core. The gameplay isn't there yet, but you as a player need to jump on
+furnitures, collect coins and fight lava/fire based enemies with your water gun.
+
+You can follow the progress with updates [here][updates]
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-014/index.md:74]:
```suggestion
You can follow the progress with updates [here][updates].
```

**Review by 17cupsofcoffee — APPROVED** [review]
Tiny grammar nitpick, but LGTM other than that.

**Review by ozkriff — APPROVED** [review]
:fire: (Made a few style tweaks)

**Comment by ozkriff** [comment]
Merged, thanks for the PR! :)

**Pull request merged**